viaf.py

```diff
@@ -868,11 +868,15 @@ def get_data(self, filename):
         data = open(path).read()
         return data
 
-
     def lookup_by_viaf(self, viaf, working_sort_name=None,
                        working_display_name=None, do_get=None):
         xml = self.get_data("mindy_kaling.xml")
         return self.parser.parse(xml, working_sort_name, working_display_name)
 
-
-
+    def lookup_by_name(self, sort_name, display_name=None, do_get=None,
+                       known_titles=None):
+        def do_get(*args, **kwargs):
+            return (200, {}, self.get_data("mindy_kaling.xml"))
+        return super(MockVIAFClient, self).lookup_by_name(
+            sort_name, display_name, do_get, known_titles
+        )
```

Comment:
This code was copied from test_viaf.py, where it's used in the context of a specific test. There should be a more useful mock VIAF client that lets us do simpler VIAF tests, but I don't have the attention span to do that work.